_about/notes/0.5.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use new Kubernetes 1.9 mutating webhook functionality to provide automatic
+pod-level proxy injection. TODO: Doc link?
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
+
+- **RBAC**. Mixer now has a resource-based access control adapter. TBD: Doc link?
+
+- **Fluentd**. Mixer now has an adapter for [fluentd](https://www.fluentd.org) telemetry collection. TODO: Doc link?
+
+- **Stdio**. The stdio adapter now lets you log to files with support for log rotation & backup, along with a host
+of controls.
+
+## Security
+
+- **Bring Your Own CA**. There have been many enhancements to the 'bring your own CA' feature. TODO: Doc link?
+
+- **PKCS8**. Add support for PKCS8 keys to Istio PKI.
+
+## Other
+
+- **Component Logging**. Istio components now offer a rich set of command-line options to control local logging, including
+common support for log rotation. 
+
+- **Consistent Version Reporting**. Istio components now offer a consistent command-line interface to report their version information.
+
+- **Optional Instance Fields**. Within configuration, definitions of Mixer instances no longer need to include every field of the
+associated template. Omitted fields get a zero or empty value.
+
```

Comment:
Does it make sense to have a known issue section?  I think we should, especially for things that will work previously but it doesn't now.   I have got people complained to me when 0.4 goes out that we didn't warm people for ports change.

Reply:
Sounds reasonable. Any recommendations for content?

Reply:
helm install is broken is one of them.  auto injection doesn't work on 1.7 or 1.8 is another one.

Reply:
Done.

Reply:
Done

Reply:
I believe mesh expansion is also broken. @costinm can provide more detail.